openedx/features/enterprise_support/api.py

```diff
@@ -441,7 +444,9 @@ def get_enterprise_consent_url(request, course_id, user=None, return_to=None, en
     else:
         return_path = reverse(return_to, args=(course_id,))
 
+    enterprise_learner_details = get_enterprise_learner_data(request.site, user)
     url_params = {
+        'enterprise_customer_uuid': enterprise_learner_details[0]['enterprise_customer']['uuid'],
```

Comment:
Since this is just an addition, I don't think we'll have to do any follow up work, as compared to ecommerce (https://github.com/edx/ecommerce/pull/1494/files#r142020271).